.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+portfolio-text-copy.txt
```

Comment:
if this wont ever be in the project best to remove it and host it somewhere else in your laptop